libs/core/src/lib/select/select.interface.ts

```diff
@@ -11,8 +11,8 @@ export const SELECT_COMPONENT = new InjectionToken<string[]>('SelectInterface');
  */
 export interface SelectInterface {
     mobileConfig: MobileModeConfig;
```

Comment:
You dont want to push this code into select, as I have another PR in Progress

Reply:
Hi Frank, we want to get rid of circular dependency for 0.19.0 version, which will be released soon. After it's done we can overwrite those changes by your changes.

Reply:
hmmm. ok.. 

My goal was push my changes latest today..